src/components/CustomInput.tsx

```diff
@@ -1,14 +1,16 @@
 import React from "react";
-import style from "./TextInput.module.scss";
+import style from "./CustomInput.module.scss";
 
-type TextInputProps = {
+type CustomInputProps = {
+  type: string;
   label: string;
   id: string;
   value: string;
   setValue: (value: string) => void;
 };
 
-export const TextInput: React.FC<TextInputProps> = ({
+export const CustomInput: React.FC<CustomInputProps> = ({
+  type,
```

Comment:
You can default it to `text` which would save you from writing it in 90% of cases

```suggestion
  type = "text",
```

Reply:
true, solved